README.md

```diff
@@ -8,20 +8,138 @@ Please sign up through [this form](https://forms.gle/1w8umSbTF4JTPwyj9) if you h
 
 **Can't wait to have you officially join the team!**
 
```

Comment:
```suggestion
We are currently localizing handbooks and articles in two ways:

1. Traditional translation process: As a collaborator, you can pick an article, translate it, and submit it for review.

2. Post-editing process: As a collaborator, you can pick an AI-translated handbook or lengthy article to post-edit, ensuring it's grammatically correct and fixing any possible AI errors, such as untranslated words or paragraphs, missing content, or incorrectly wrapped paragraphs in backticks.

In both localization methods, the collaborator's meticulous revision is crucial, especially on AI-translated handbooks and articles, since AI translations are known to make mistakes or miss certain details.

Please take a look and decide which of these two methods you would like to collaborate on.

## Article Translation Process

### Pick Your Article
Once you're in as a Collaborator, it's time to choose which article you'd like to work on. Check out the list of available issues/articles and pick the one that speaks to you from the Project of the spoken language you would like to help on.

### Claim Your Article
Found the perfect piece to translate? Awesome! Just comment /translate on the issue. This will automatically assign the issue to you and move the card to the "in Translation" status.

*It may take a few minutes before the GitHub Actions workflow responds to the command. When it's done, it will add a comment "@username We have assigned this article to you."

### Ready for Review
After you've translated your piece, give us a heads up by commenting /review. This will automatically move the card to the "in Review" status and let our Language Leads or Proofreaders know that it's time to revise your translation and prepare before potentially publishing it.

```

Reply:
this is missing from the merged PR, it doesn't look like the PR was ready for merging @sidemt